third_party/closure-compiler/runner/src/AmpCommandLineRunner.java

```diff
@@ -0,0 +1,34 @@
+import java.util.List;
+
+import com.google.javascript.jscomp.CommandLineRunner;
+import com.google.javascript.jscomp.CompilerOptions;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Sets;
+
+public class AmpCommandLineRunner extends CommandLineRunner {
+
+  private static final List<String> LOG_STRIP_SUFFIX = ImmutableList.of(
+      "dev.fine",
+      "dev.assert");
+
+  protected AmpCommandLineRunner(String[] args) {
+    super(args);
+  }
+
+  @Override
+  protected CompilerOptions createOptions() {
+    CompilerOptions options = super.createOptions();
+    options.setStripTypes(Sets.newHashSet(LOG_STRIP_SUFFIX));
```

Comment:
so ive been researching it some more and its not that its running after optimization, its that it runs against the es5 -> es6 transpiled code meaning its walking through something that looks like the screenshot below (where its already been prefixed with the namespace):

<img width="541" alt="screen shot 2016-03-17 at 1 17 30 am" src="https://cloud.githubusercontent.com/assets/354746/13840215/2e5c32f0-ebde-11e5-851b-4f67b3163aee.png">